chapter_linear-networks/linear-regression.md

```diff
@@ -342,7 +342,7 @@ rather than writing costly for-loops in Python.**)
 %matplotlib inline
 from d2l import mxnet as d2l
 import math
-import numpy as np
+from mxnet import np
```

Comment:
Can we keep using import numpy as np like what we do for pytorch and tf? This will simplify the implementation (tab all) and we don't need any change in this section

Reply:
Same reason as https://github.com/d2l-ai/d2l-en/pull/2078#discussion_r832814639